components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -89,7 +90,7 @@
 	
 	private static final Log log = LogFactory.getLog(APIProviderImpl.class);
```

Comment:
Thanks for the comment. I have changed the code and in the commit 75ad008cf0b14b14f74309690e5386afce48bf19